<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMain">
   <item row="0" column="0">
    <widget class="QPushButton" name="ui_btn_bakeMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prepares the scene for baking.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>ENTER BAKING MODE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string>Render</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ui_chb_bakeLightMap">
       <property name="text">
        <string>Lightmap</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ui_chb_bakeAO">
       <property name="text">
        <string>AO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ui_chb_bakeDirectionalLM">
       <property name="text">
        <string>Directional LM</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ui_btn_BakeLightmap">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>BAKE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>ui_lst_lmLayers</zorder>
  <zorder>ui_btn_BakeLightmap</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>ui_btn_bakeMode</zorder>
  <zorder>ui_chb_bakeDirectionalLM</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
